save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt

```diff
@@ -132,6 +133,9 @@ val createVulnerabilityView: VFC = VFC {
         "Base Score Calculator".t(),
         bodyBuilder = {
             cvssBaseScoreCalculator {
+                baseMetricsOfVulnerability = cosv.severity?.let { severityList ->
```

Comment:
can be simplified: `cosv.severity?.singleOrNull { it.type == SeverityType.CVSS_V3 }?.let { CvssVectorV3(it.score.parsingVectorToMap()).baseMetrics }`